lib/rdkafka/admin/create_acl_report.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Rdkafka
+  class Admin
+    class CreateAclReport
+
+      # Upon successful creation of Acl RD_KAFKA_RESP_ERR_NO_ERROR - 0 is returned as rdkafka_response
+      # @return [Integer]
+      attr_reader :rdkafka_response
+
+
+      # Upon successful creation of Acl emtpy string will be returned as rdkafka_response_string
```

Comment:
```suggestion
      # Upon successful creation of Acl empty string will be returned as rdkafka_response_string
```

Reply:
@geoff2k sorry about the typos. fixed. I am combining all the code review comments in a single commit.